easycv/datasets/loader/build_loader.py

```diff
@@ -58,9 +59,10 @@ def build_dataloader(dataset,
     Returns:
         DataLoader: A PyTorch dataloader.
     """
+    seed = sync_random_seed(seed)
+    rank, world_size = get_dist_info()
```

Comment:
move to line 66, should be executed only when dist is True

Reply:
done